convertJSONScript.js

```diff
@@ -0,0 +1,42 @@
+const { Client } = require('pg');
+const createData = require('./lib/createData');
+const parse = async (id,parsed)=> {
+  try {
+    let SQL = '';
+    const { fieldNames, values } = createData(parsed);
+    for (let i = 0; i < fieldNames.length; i++) {
+      SQL = SQL + `UPDATE companies SET ${fieldNames[i]}='${values[i]}' WHERE id=${id};`;
```

Comment:
I'm wondering if this gives us a SQL vulnerability - could we use the $1 positional approach so that the values get escaped for us?